dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/add-fees-to-an-existing-payment-group/add-fees-to-an-existing-payment-group.event-handler.ts

```diff
@@ -0,0 +1,90 @@
+import { EntityManager } from 'typeorm';
+import { DbRequestSource, FeeRecordEntity, PaymentEntity, UtilisationReportEntity } from '@ukef/dtfs2-common';
+import { BaseUtilisationReportEvent } from '../../event/base-utilisation-report.event';
+import { FeeRecordStateMachine } from '../../../fee-record/fee-record.state-machine';
+import { feeRecordsMatchAttachedPayments } from '../helpers';
+
+type AddFeesToAnExistingPaymentGroupEventPayload = {
+  transactionEntityManager: EntityManager;
+  feeRecordsToAdd: FeeRecordEntity[];
+  otherFeeRecordsInPaymentGroup: FeeRecordEntity[];
```

Comment:
Rather than `otherFeeRecordsInPaymentGroup` I think this should be `existingFeeRecordsInPaymentGroup`